test/smoke/FirefoxPrivateVPNUITest/FirefoxPrivateVPNUITest/Sessions/BrowserSession.cs

```diff
@@ -38,16 +38,14 @@ public BrowserSession()
                 catch (Exception)
                 {
                     // 1. Creating a Desktop session
-                    DesiredCapabilities desktopAppCapabilities = new DesiredCapabilities();
-                    desktopAppCapabilities.SetCapability("app", "Root");
-                    var desktopSession = new WindowsDriver<WindowsElement>(new Uri(WindowsApplicationDriverUrl), desktopAppCapabilities);
+                    var desktopSession = new DesktopSession();
                     bool retry = true;
                     int retryTimes = 0;
                     IWebElement firefoxWindows = null;
                     while (retry)
                     {
                         retryTimes += 1;
-                        WebDriverWait wait = new WebDriverWait(desktopSession, TimeSpan.FromSeconds(600));
+                        WebDriverWait wait = new WebDriverWait(desktopSession.Session, TimeSpan.FromSeconds(60));
```

Comment:
It looks like we have some of these wait periods throughout our tests.  Have they been implemented with a wait of x amount of time regardless of the result?  Should we look at having the wait simply be a timeout (max amount of time to wait) instead, so that the test can progress if a condition is met earlier than the max wait time.

Reply:
If the condition is met earlier, the program wont wait that long.